src/lib/DropDownController.js

```diff
@@ -0,0 +1,35 @@
+class DropDownController {
+  handleEvent(event) {
+    if (event.type === 'click' && event.target !== this.elem) {
+      this.hideDropDown();
+    }
+  }
+
+  triggerDropDown(elem) {
+    if (this.elem === elem) {
+      this.elem = null;
+      elem.classList.toggle('expanded', false);
```

Comment:
Replace with `this.hideDropDown();`. The listener needs to be removed.

And perhaps the method should be renamed to `toggleDropDown` since you're toggling?

Reply:
Yes, I have replaced this part with `this.hideDropDown()` method and renamed the `triggerDropDown` method to `toggleDropDown`